ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -83,6 +83,54 @@ function deleteDescriptionFromJSONSchema(jsonSchema) {
   return { ...jsonSchema, description : "" };
 }
 
+function camelCaseToCapitalize(text){
+  const result = text.replace(/([A-Z])/g, " $1");
+
+  return result.charAt(0).toUpperCase() + result.slice(1);
+}
+
+function addTitleToPropertiesJSONSchema(jsonSchema) {
+  const newProperties = jsonSchema?.properties
+
+  if (newProperties && typeof newProperties === 'object'){
+    Object.keys(newProperties).map(key => {
+      if (Object.prototype.hasOwnProperty.call(newProperties, key)){
+        let defaultValue;
+        let types = []
+        if (!Array.isArray(newProperties[key].type) && Object.prototype.hasOwnProperty.call(newProperties[key], 'type')){
+          types.push(newProperties[key].type)
+        } else {
+          types.push(...newProperties[key].type)
+        }
+        if (types.includes('null')){
+          defaultValue = null
+        } else if (types.includes('integer')){
+          defaultValue = 0
+        } else if (types.includes('string')){
+          defaultValue = ''
+        } else if (types.includes('array')){
+          defaultValue = []
+        }
+        newProperties[key] = {
+          ...newProperties[key],
+          title : camelCaseToCapitalize(key),
+          default : defaultValue
+        }
+        // if (typeof newProperties[key] === 'object' && Object.prototype.hasOwnProperty.call(newProperties[key], 'properties')){
```

Comment:
Including a set of predefined values for null values is a good idea. My sense is that this should be reused across other pattern-based components (filters, apps). Understand that the entire /management UIs will convert to pattern-style UIs, so making these reusable and centralizing this list will be worth the effort.